specification/potato.md

```diff
@@ -0,0 +1,3 @@
+## Potato
+
+No potato was harmed during the production of this specification. We thank potato for its contribution in the creation of this specification.
```

Comment:
```suggestion
No starchy root vegetables were harmed during the production of this specification. We thank potato for its contribution in the creation of this specification.
```
Minimizing use of term in its own description. Very serious.

Reply:
Totally doing this. Added out of this commit however because we changed a few things.

Reply:
@aqu-erp check it out now